app/steps/ui/executors/named/template.html

```diff
@@ -0,0 +1,96 @@
+{% extends "includes/two_thirds_form.html" %}
+
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "widgets/checkanswer.njk" import checkanswer %}
+{% from "widgets/lists.html" import unorderedList %}
+
+{% block form_content %}
+    {% call govukFieldset({
+        legend: {
+            text: content.title | safe,
+            classes: "govuk-fieldset__legend--l",
+            isPageHeading: true
+        }
+    }) %}
+
+
+<pre>{{ fields | dump | safe }}</pre>
+
```

Comment:
The for loop below is existing behaviour for displaying all the executors but since the logic for entering the exec name now takes place in /executor-names this needs the be worked.